src/help/components/contextual/styles/contextual-help.scss

```diff
@@ -9,3 +11,15 @@
     vertical-align: super;
   }
 }
+
+// Note: this isn't in a module because of the
+// franklin overrides. can be moved to modules
+// if sliding panel heading colour can be specified
+.contextual-help-panel {
```

Comment:
You can use `:global` for this. See usage in the rest of the codebase

Reply:
There's no way allowing having `[data-article-id] { ...}` in the css module is there?

Reply:
CSS modules should only transform classnames